server/utils/nunjucksSetup.ts

```diff
@@ -29,7 +29,9 @@ export default function nunjucksSetup(app: express.Express): void {
       '/cookies-policy',
       '/privacy-policy',
     ]
-    const externalUser = externalUrls.some(externalUrl => req.url.startsWith(externalUrl))
+    const externalUser =
+      externalUrls.some(externalUrl => req.url.startsWith(externalUrl)) ||
+      req.hostname.toLowerCase().startsWith('send-legal-mail-to-prisons')
```

Comment:
This is the line that cannot be tested properly other than by deploying and seeing what happens 🤞